### ISSUE TITLE
pikaur -Qnq | pikaur -S - does nothing/ stdin?

### ISSUE THREAD

**Issue by T35R6braPwgDJKq** [actionless/pikaur#587]
The following command does not work
```pikaur -Qnq | pikaur -S -
:: Proceed with installation? [Y/n] 
:: [v]iew package details   [m]anually select packages
>> % 
```

```
pikaur -Qqn | sudo pacman -S -
```
does though

**Comment by actionless** [comment]
it was once fixed https://github.com/actionless/pikaur/issues/411#issuecomment-544309343

but aparently it regressed at some point, gonna look into it later

**Comment by actionless** [comment]
ah, nope, in that ticket it was using `--noconfirm` - apparently it was never implemented

**Comment by T35R6braPwgDJKq** [comment]
"Vim: Warning: Input is not from a terminal"

**Comment by T35R6braPwgDJKq** [comment]
> `$ echo mpv | pikaur -Sw -` works for me

well, it would help if you used the same commands a me, the one with the errors...... `pikaur -Qqm | pikaur -Sw -`

**Comment by actionless** [comment]
read carefully the issue template, what information need to be provided, i'm not gonna to continue arguing about nothing

**Comment by T35R6braPwgDJKq** [comment]
yeah, you already closed the issue right after you thought you fixed it.. I gave the needed information. Your job is simply to reproduce which is possible by executing the command. SInce you are - as always it seems - not willing to, i dont give a damn anymore.
